solr/solr-ref-guide/modules/deployment-guide/pages/solrj.adoc

```diff
@@ -95,11 +95,11 @@ Requests are sent in the form of {solr-javadocs}/solrj/org/apache/solr/client/so
 
 `SolrClient` has a few concrete implementations, each geared towards a different usage-pattern or resiliency model:
 
-- {solr-javadocs}/solrj/org/apache/solr/client/solrj/impl/Http2SolrClient.html[`Http2SolrClient`] - a general purpose client based on Jetty HttpClient.  Supports HTTP/2 and HTTP/1.1, async, non-blocking.  Most used & tested.
+- {solr-jetty-javadocs}/solrj/org/apache/solr/client/solrj/impl/Http2SolrClient.html[`Http2SolrClient`] - a general purpose client based on Jetty HttpClient.  Supports HTTP/2 and HTTP/1.1, async, non-blocking.  Most used & tested.
```

Comment:
[0] I'm still a little confused about whether the new "jetty" module is opt-in or not.  But if it's opt-in, maybe we should lead with the "JDK" clients and have the Jetty clients in a separate list (or at the end of this list).

[-0] If we're adding a new CloudJettySolrClient, it should probably be mentioned here, right?